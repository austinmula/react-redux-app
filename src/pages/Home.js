import Heading from "../components/Heading";

export default function Home() {
  return (
    <div>
      <h1 data-testid="heading">This is the homepahge</h1>
      <Heading title="Heading" text="This is simple text for testing" />
    </div>
  );
}
