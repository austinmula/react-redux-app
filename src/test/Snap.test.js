import React from "react";
import { Provider } from "react-redux";
import { BrowserRouter } from "react-router-dom";
import { store } from "../app/store";
import renderer from "react-test-renderer";
import Home from "../pages/Home";

test("snapshot test works", () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <Home />
        </BrowserRouter>
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
