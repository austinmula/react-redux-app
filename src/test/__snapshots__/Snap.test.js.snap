// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test works 1`] = `
<div>
  <h1
    data-testid="heading"
  >
    This is the homepahge
  </h1>
  <div>
    <h3>
      Heading
    </h3>
    <p>
      This is simple text for testing
    </p>
  </div>
</div>
`;
